pkg/test/dependency/mixer.go

```diff
@@ -29,14 +39,28 @@ func (a *mixer) String() string {
 	return "mixer"
 }
 
-func (a *mixer) Initialize() (interface{}, error) {
-	return nil, nil
+func (a *mixer) Initialize(env environment.Interface) (interface{}, error) {
+	if c, ok := env.(cluster.Internal); ok {
```

Comment:
correct me if I'm wrong, but I suspect that this block won't work correctly ATM, since both environments implement both `Internal` interfaces due to the fact that both interfaces are empty. Perhaps we should add a dummy method to both `Internal` interfaces to differentiate them?

Reply:
PTAL.

Yes it will (I actually have follow up CLs locally that start filling these in,  I just had the cut-off here for PR here). Added the dummy method, just in case.

 I've fixed formatting and some of the linter issues. The rest seemed unused fields, which is also a point in time thing. There are many of them. Some of it will be resolved naturally as we make progress.